test cases/common/223 custom target input extracted objects/verify.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import sys, os
+
+if __name__ == '__main__':
+    if len(sys.argv) != 2:
+        print(sys.argv[0], 'output')
+        sys.exit(1)
+    with open(sys.argv[1], 'rb') as resultFile:
+    	result = resultFile.read().decode('UTF8')
```

Comment:
__[Flake8]__ [E101] indentation contains mixed spaces and tabs ([link](https://sider.review/gh/repos/19784232/pulls/5695#issue-4431738))

<sub>posted by Sider</sub>

Reply:
Fixed in e5093a7c